Quintor/src/main/java/com/example/quintor/DashboardController.java

```diff
@@ -90,9 +93,9 @@ public void uploadButtonAction() {
         if (f != null) {
             try {
                 if (uploadFile(f, userId)) {
-                    System.out.println("gelukt");
+                    uploadToPostgres(f, userId);
+
                 } else {
```

Comment:
Er moet denk ik wel iets gebeuren als het uploaden niet gelukt is. Moeten we even overleggen